packages/@cdktf/provider-generator/lib/get/__tests__/provider.test.ts

```diff
@@ -7,7 +7,7 @@ import {
 const getProvider = (constraint: TerraformDependencyConstraint) =>
   expectImportMatchSnapshot(constraint);
 
-getProvider(new TerraformProviderConstraint("aws@= 2.60.0"));
+getProvider(new TerraformProviderConstraint("aws@= 3.61.0"));
```

Comment:
For everything besides this test, I think it may make sense to use `~> 3.0` instead. That better matches the provider documentation.

There is an argument to be made that since the examples are used as extended integration test, that they shouldn't change arbitrarily; however, I'd rather know sooner than later about a new AWS provider breaking something.